Packages/Account/Sources/Account/AccountDetailView.swift

```diff
@@ -400,6 +418,10 @@ extension View {
   }
 }
 
+class BlockConfirmation: ObservableObject {
+  @Published var isPresentingBlockingConfirmation: Bool = false
+}
```

Comment:
This should be directly in the view as a simple State property and passed to the context menu as a binding. 
`ObservableObject` is also not used anymore as the project was migrated to the new observation framework.